src/main/scala/mesosphere/mesos/RunSpecOfferMatcher.scala

```diff
@@ -9,12 +10,14 @@ import org.apache.mesos.Protos.Offer
 import org.slf4j.LoggerFactory
 
 import scala.collection.immutable.Seq
+import scala.concurrent.duration._
 
 object RunSpecOfferMatcher {
 
   val log = LoggerFactory.getLogger(getClass)
 
-  def matchOffer(runSpec: RunSpec, offer: Offer, otherInstances: => Seq[Instance], givenAcceptedResourceRoles: Set[String]): ResourceMatchResponse = {
+  def matchOffer(runSpec: RunSpec, offer: Offer, otherInstances: => Seq[Instance],
+    givenAcceptedResourceRoles: Set[String], drainingTime: FiniteDuration)(implicit clock: Clock): ResourceMatchResponse = {
```

Comment:
IMO, given that the parameter has a default value, we shouldn't set default value for drainingTime in other places, even if the PR gets noisy.

But I would be happy to change it if you think that's better